docs/catalog/cloud/Design for independent scaling of logical components.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- cloud
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Scale logical components independently
+
+## Description
+A microservice architecture may reduce the compute resources required, as it allows each independent component to be scaled according to its own demand.
+
+## Solution
+
+Use cloud native projects like [Dapr](https://dapr.io) Framework to help you separate your application functionality into different microservices, to allow independent scaling of its logical components.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, scaling logical components independently will impact:
+
+- `E`: By optimizing the scaling of the most consuming application components, we reduce the amount of energy consumed by the application globally.
```

Comment:
I would assume that resources are not used, because we scale down in low load scenarios -> no embodied carbon emissions for this application
(but I'm not the author)

Reply:
Agree. E is also affected. I will leave it like that